src/View/Helper/FormHelper.php

```diff
@@ -102,6 +109,11 @@ public function __construct(View $View, array $config = [])
 
         $this->_defaultWidgets = $this->_widgets + $this->_defaultWidgets;
 
+        $this->_controlMethod = 'input';
```

Comment:
You could also default the property to "input" - and only overwrite here if necessary.
That would get rid of this line.